src/components/LeaderboardPage/LeaderboardPage.docs.mdx

```diff
@@ -0,0 +1,32 @@
+# LeaderboardPage
```

Comment:
The Storybook docs content is written in Spanish. The project's rule requires code and comments to be in English for consistency and to help cross-team collaboration.

**Sugerencia:**
```typescript
Translate the docs to English. Example header: '# LeaderboardPage' and then describe props/behavior in English. Keep the same sections (Props, Data shape, Behavior) but use English text.
```